src/utilities/aspect-ratio/demo/youtube-video.twig

```diff
@@ -0,0 +1,16 @@
+{% if aspectRatio %}
+  {% set style = (style|default('')) ~ ';--aspect-ratio:' ~ aspectRatio %}
+{% endif %}
+
+<iframe
+  class="{{ aspectRatioClass }} u-border-medium"
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/Ty5jtMZXbmk"
+  allowfullscreen
+  style="width: 100%;
```

Comment:
What happens if this `width` `style` is omitted?

Reply:
I adjusted the CSS and removed this `style` `width` attribute. The intent is that it should work without it.